tools/biobb_pytorch/.shed.yml

```diff
@@ -0,0 +1,20 @@
+name: biobb_pytorch
+owner: chemteam
+description: "biobb_pytorch is the Biobb module collection to create and train ML & DL models using the popular [PyTorch](https://pytorch.org/) Python library."
+homepage_url: https://github.com/bioexcel/biobb_pytorch
+long_description: |
```

Comment:
can you convert this file into a suite-based file? https://github.com/galaxyproject/tools-iuc/blob/main/tools/semibin/.shed.yml